assets/src/edit-story/app/media/media3p/attribution.js

```diff
@@ -80,17 +80,30 @@ const coverrUrl =
 const tenorUrl =
   'https://tenor.com?utm_source=web_stories_wordpress&utm_medium=referral';
 
+const MEDIA_PROVIDER = {
+  coverr: 'Coverr',
+  tenor: 'Tenor',
+  unsplash: 'Unsplash',
+};
+const getAriaLabel = (provider) =>
+  sprintf(
+    /* translators: %s: media provider name. */
+    __('Powered by %s', 'web-stories'),
+    MEDIA_PROVIDER[provider]
```

Comment:
I feel like we could just pass the provider name in instead of making this dictionary. If somebody misspells the property name then this returns `undefined`

Maybe we just do this?
```suggestion
const MEDIA_PROVIDER = {
  coverr: 'Coverr',
  tenor: 'Tenor',
  unsplash: 'Unsplash',
};
const getAriaLabel = (provider) =>
  sprintf(
    /* translators: %s: media provider name. */
    __('Powered by %s', 'web-stories'),
    provider
```

And then when we're passing the values in we could always do `MEDIA_PROVIDER.Coverr` and stuff?

Reply:
Thanks, @samwhale - should be better now. This was overkill on my part 🙃